recipe_scrapers/__init__.py

```diff
@@ -286,6 +286,7 @@
 from .woop import Woop
 from .yemek import Yemek
 from .yummly import Yummly
+from .zaubertopfde import Zaubertopfde
```

Comment:
I think we should use the classname `ZauberTopf` for this one; the `.de` in the domain-name isn't intended to be part of the property/publication name.